lua/carbon-now.lua

```diff
@@ -1,7 +1,9 @@
 -- module represents a lua module for the plugin
+local types = require("types")
 local carbon = {}
 
--- default configs
+-- default config
+---@type cn.ConfigSchema
```

Comment:
this doesn't exist anymore right?

Reply:
The `cn.lua` does not exist anymore. Types however, are packed within that 'alias'.